MAUI/TreeMap/getting-started.md

```diff
@@ -0,0 +1,571 @@
+                new PopulationDetails() { Continent ="North America", Country = "Guatemala", Population = 18092026 },
+                new PopulationDetails() { Continent ="South America", Country = "Bolivia", Population = 12388571 },
+                new PopulationDetails() { Continent ="North America", Country = "Honduras", Population = 10593798 },
+                new PopulationDetails() { Continent ="North America", Country = "Nicaragua", Population = 7046311 },
+                new PopulationDetails() { Continent ="South America", Country = "Paraguay", Population = 6861524 },
+                new PopulationDetails() { Continent ="North America", Country = "El Salvador", Population = 6364943 },
+                new PopulationDetails() { Continent ="North America", Country = "Costa Rica", Population = 5212173 },
+                new PopulationDetails() { Continent ="South America", Country = "Uruguay", Population = 3423109 },
+            };
+    }
+    public ObservableCollection<PopulationDetails> PopulationDetails
+    {
+        get;
+        set;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+#### Bind data source for TreeMap
+
+You can use `DataSource` property of SfTreeMap to populate the treemap items. The `PrimaryValuePath` specifies the name of the property in the data object that provides the primary value used to determine the size of each item in the tree map. The primary value typically represents the main quantitative data associated with each item.
+
+{% tabs %}
+{% highlight XAML hl_lines="3 5 6 7" %}
+
+<ContentPage 
+    xmlns:local="clr-namespace:SfTreeMapSample"
+    xmlns:treemap="clr-namespace:Syncfusion.Maui.TreeMap;assembly=Syncfusion.Maui.TreeMap">
+    <treemap:SfTreeMap x:Name="treeMap"
+                    DataSource="{Binding PopulationDetails}"
+                    PrimaryValuePath="Population"
+                    RangeColorValuePath="Population">
+        <treemap:SfTreeMap.BindingContext>
+            <local:PopulationViewModel />
+        </treemap:SfTreeMap.BindingContext>
+    </treemap:SfTreeMap>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# hl_lines="10 11" %}
+
+using Syncfusion.Maui.TreeMap;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfTreeMap treeMap = new SfTreeMap();
+        PopulationViewModel viewModel = new PopulationViewModel();
+        treeMap.DataSource = viewModel.PopulationDetails;
+        this.Content = treeMap;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Add labels
+
+You can use `LabelPath` property of SfTreeMap `LeafItemSettings` to display text for the leaf items.
+
+{% tabs %}
+{% highlight XAML hl_lines="12" %}
+
+<ContentPage   
+    xmlns:local="clr-namespace:SfTreeMapSample"
+    xmlns:treemap="clr-namespace:Syncfusion.Maui.TreeMap;assembly=Syncfusion.Maui.TreeMap">
+    <treemap:SfTreeMap x:Name="treeMap"
```

Comment:
Add code snippet for tree map control alone. So remove content page and all.

Reply:
Added code snippet for tree map control alone